pipelines/ocw/pipeline-theme-assets.yml

```diff
@@ -0,0 +1,57 @@
+---
+resource_types:
+  - name: rclone
+    type: docker-image
+    source:
+      repository: mitodl/concourse-rclone-resource
+      tag: latest
+resources:
+  - name: ocw-hugo-themes
+    type: git
+    source:
+      uri: https://github.com/mitodl/ocw-hugo-themes.git
+      branch: ((hugo-theme-branch))
+  - name: ocw-artifacts
+    type: rclone
+    source:
+      config: |
+        [s3-remote]
+        type = s3
+        provider = AWS
+        env_auth = true
+        region = us-east-1
+jobs:
+  - name: build-theme-assets
+    serial: true
+    plan:
+      - get: ocw-hugo-themes
+        trigger: true
+      - task: build-ocw-www-hugo-theme
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source: {repository: mitodl/ocw-course-publisher, tag: latest}
```

Comment:
What is ocw-course-publisher?

Reply:
That's a docker image that has Node and Hugo pre-installed on it